config.py

```diff
@@ -150,16 +168,27 @@ class ProductionConfig(Config):
 
     @classmethod
     def init_app(cls, app):
-        Config.init_app(app)
+        console_handler = logging.StreamHandler(stream=sys.stdout)
+        console_handler.setLevel(logging.INFO)
+        jsonformatter = jsonlogger.JsonFormatter(
+        '%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+        console_handler.setFormatter(jsonformatter)
 
+        loggers = [app.logger, 
+                   getLogger('elasticsearch'),
+                   getLogger('redislite')]
 
+        for logger in loggers:
+            logger.addHandler(console_handler)
+            logger.setLevel(logging.INFO)
+
+        Config.init_app(app)
 
 
 
 
 config = {
     'development': DevelopmentConfig,
     'testing': TestingConfig,
-    'production': ProductionConfig,
-    'default': DevelopmentConfig
```

Comment:
wasn't default used by default is no env is specified? it make sense to keep it, so it is easy to change in a place where you see al the configuations available